### PR TITLE
nautilus: rpm: add rpm-build to SUSE-specific make check deps

### PR DESCRIPTION
**Pull request by smithfarm**
backport trackers:

* https://tracker.ceph.com/issues/43280
* https://tracker.ceph.com/issues/43288

---

backport of

* https://github.com/ceph/ceph/pull/32083
* https://github.com/ceph/ceph/pull/32153

parent trackers:

* https://tracker.ceph.com/issues/23981
* https://tracker.ceph.com/issues/42612

this backport was staged using ceph-backport.sh version 15.0.0.6950
find the latest version at https://github.com/ceph/ceph/blob/master/src/script/ceph-backport.sh

### CODE REVIEW

**Comment by yuriw** [comment]
[wip-yuri-testing-2020-01-14-2058-nautilus](https://trello.com/c/j3EAERXf/807-wip-yuri-testing-2020-01-14-2058-nautilus)

**Comment by smithfarm** [comment]
jenkins test make check

**Comment by smithfarm** [comment]
@LenzGr @capri1989 pinging again for review?

File: install-deps.sh [hunk 1]
```diff
@@ -375,7 +375,11 @@ else
     opensuse*|suse|sles)
         echo "Using zypper to install dependencies"
         zypp_install="zypper --gpg-auto-import-keys --non-interactive install --no-recommends"
-        $SUDO $zypp_install systemd-rpm-macros
+        $SUDO $zypp_install systemd-rpm-macros rpm-build || exit 1
+        if [ -e /usr/bin/python2 ] ; then
+            # see https://tracker.ceph.com/issues/23981
+            $SUDO $zypp_install python2-virtualenv python2-devel || exit 1
```
Comment (shyukri) [inline install-deps.sh:381]: @smithfarm Looks like `python2-virtualenv` is no longer available for TW.

**Pull request merged**